packages/babel-generator/src/generators/expressions.js

```diff
@@ -22,6 +22,64 @@ export function DoExpression(node: Object) {
   this.print(node.body, node);
 }
 
+export function ArrayMatchPattern(node: Object) {
+  this.token("[");
+  this.printList(node.children, node);
+  if (node.hasRest) {
+    this.token("...");
+  }
+  if (node.restIdentifier !== null) {
+    this.print(node.restIdentifier, node);
+  }
+  this.token("]");
+}
+
+export function MatchExpression(node: Object) {
+  this.word("match");
+  this.token("(");
+  this.print(node.expression);
+  this.token(")");
+  this.space();
+  this.token("{");
+  this.newline();
+  this.indent();
+  this.printList(node.clauses, node);
+  this.dedent();
+  this.token("}");
+}
+
+export function MatchClause(node: Object) {
+  this.print(node.pattern, node);
+  this.token("=>");
+  this.space();
+  this.print(node.body, node);
+  this.newline();
+}
+
+export function MatchGuard(node: Object) {
+  this.workd("if");
```

Comment:
Seems to be a typo here, should this be `word` instead of `workd`?

Reply:
Yep, it's a mistake, thx